bin/generate.py

```diff
@@ -1261,18 +1275,30 @@ def parse(key, name, yaml, parent):
                 fatal(f'Could not parse {parent.path / name} as a testcase or directory.')
 
             if is_testcase(yaml):
+                count = 1
+                if yaml is not None and 'count' in yaml and isinstance(yaml['count'], int):
+                    count = yaml['count']
+
                 if not config.COMPILED_FILE_NAME_REGEX.fullmatch(name + '.in'):
                     error(f'Testcase \'{parent.path}/{name}.in\' has an invalid name.')
                     return None
 
-                # If a list of testcases was passed and this one is not in it, skip it.
-                if not self.process_testcase(parent.path / name):
-                    return None
+                ts = []
+                for i in range(count):
+                    # If a list of testcases was passed and this one is not in it, skip it.
+                    if not self.process_testcase(parent.path / name):
+                        continue
+
+                    curname = name if count == 1 else name + f'-{i+1:0{len(str(count))}}'
+
+                    t = TestcaseRule(self.problem, self, key, curname, yaml, parent, i)
+                    if t.path in self.known_cases:
+                        error(f'{t.path} was already parsed')
```

Comment:
Maybe clarify this a bit; something like `Counted testcase name already appears in generators.yaml`

Reply:
The error was there before and we also display them as if it would have been multiple rules?